dz6 part2/dz6 part2/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace dz6_part2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            string MyWord = "кратер";
+            Console.WriteLine(MyWord.Remove(1,1));
+            Console.WriteLine(MyWord.Remove(0, 5)+ MyWord.Remove(5, 1).Remove(0, 4) + MyWord.Remove(3, 3).Remove(1, 1));
```

Comment:
Локальные в C# переменные принято называть с маленькой буквы/
```suggestion
            string myWord = "кратер";
            Console.WriteLine(myWord.Remove(1,1));
            Console.WriteLine(myWord.Remove(0, 5)+ myWord.Remove(5, 1).Remove(0, 4) + myWord.Remove(3, 3).Remove(1, 1));
```